chore: contributions github (#2)
sinharaksh1t/rust-boilerplate-starship: .github/contribution-guide.md

### Thank You!

> I would like to Thank You, for considering contributing to this tiny module. I will be very grateful for your contribution.

Reviewer: ```suggestion
We would like to thank you for considering to contribute to this module. We are forever grateful to our contributors.
```

> Important: Following these guidelines help to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.

Reviewer: ```suggestion
Important: Following these guidelines help to communicate that you respect the time of the developers developing and maintaining this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
```

#### Pull Request

> - There is a pull request template, which needs to be filled for any pull request. It will be automatically created once you create a pull request, but you go through it here [PR-template](./PULL_REQUEST_TEMPLATE.md).

Reviewer: ```suggestion
- There is a pull request template that needs to be filled for any pull requests. It will automatically be generated once you create a pull request. Feel free to go through it [here](./PULL_REQUEST_TEMPLATE.md).
```
